src/components/signup/SignupState.js

```diff
@@ -509,6 +509,8 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
         //   }),
       ])
 
+      fireSignupEvent('SUCCESS')
```

Comment:
use constants and we said it should be called SIGNUP_SUCCESS, keep same format as other events in signup

Reply:
keep the same format as other events in signup - it is exactly what I did.
take a look at the fireSignupEvent method and how it is used inside SignupSrate.js